dotnet-client-libraries/Selenium-JavaScript/Tests/JavaScriptTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using FluentAssertions;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.WaitHelpers;
+
+namespace Tests;
+public class JavaScriptTests : IClassFixture<WebFactory>, IDisposable
+{
+    private readonly IWebDriver _driver;
+
+    private readonly string _webUrl = "https://localhost:5055";
+
+    public JavaScriptTests(WebFactory factory)
+    {
+        factory.HostUrl = _webUrl;
+        factory.CreateDefaultClient();
+
+        _driver = new ChromeDriver();
+    }
+
+    [Fact]
```

Comment:
@01vadim10 
Can we remove th `LiveTest` suffix on the test names

Reply:
@elfico 
But according to the Guidelines and initial comment to this PR in Jira I should add `LiveTest` at the end of my test name because I'm using `ChromeDriver`
![image](https://github.com/CodeMazeBlog/CodeMazeGuides/assets/1189966/1cabcc13-b384-4539-95ce-a1e78f087ee6)
![image](https://github.com/CodeMazeBlog/CodeMazeGuides/assets/1189966/0832a3e7-f82f-4cb2-97a5-5ad4e4a16dc2)